From dcdffbce2ed8467a8c7d82c321c5cd63b589a4b6 Mon Sep 17 00:00:00 2001
From: Jonathan Nieder <jrnieder@gmail.com>
Date: Sun, 28 Nov 2010 20:47:07 +0800
Subject: [BUILTIN] Stop documenting EXSHELLPROC

At some point between ash 0.3.5-11.0.1 and ash 0.3.8-37, Debian
ash stopped using the EXSHELLPROC exception to handle shell
scripts without a magic number.

Remove all remaining references to it to avoid confusion.

Signed-off-by: Jonathan Nieder <jrnieder@gmail.com>
Signed-off-by: Herbert Xu <herbert@gondor.apana.org.au>
(cherry picked from commit 8393af1b1412201a2815e0a21cfec1e7d2f83b68)

[Excluding src/TOUR from the diff --- src/TOUR is not part of upstream
tarballs for some reason.]

Signed-off-by: Jonathan Nieder <jrnieder@gmail.com>
---
 ChangeLog   |    1 +
 src/error.h |    1 -
 2 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/ChangeLog b/ChangeLog
index 207ff29..a3ffd3e 100644
--- a/ChangeLog
+++ b/ChangeLog
@@ -1,6 +1,7 @@
 2010-11-28  Jonathan Nieder <jrnieder@gmail.com>
 
 	* Use EXEXIT in place of EXEXEC.
+	* Stop documenting EXSHELLPROC.
 
 2010-11-28  Gerrit Pape <pape@smarden.org>
 
diff --git a/src/error.h b/src/error.h
index ad1e55b..f236d9f 100644
--- a/src/error.h
+++ b/src/error.h
@@ -66,7 +66,6 @@ extern int exception;
 /* exceptions */
 #define EXINT 0		/* SIGINT received */
 #define EXERROR 1	/* a generic error */
-#define EXSHELLPROC 2	/* execute a shell procedure */
 #define EXEXIT 4	/* exit the shell */
 
 
-- 
1.7.4.1

